src/car/dto/create-car.dto.ts

```diff
@@ -0,0 +1,51 @@
+import { CajaType } from '@prisma/client';
+import {
+  IsBoolean,
+  IsEnum,
+  IsInt,
+  IsNotEmpty,
+  IsPositive,
+  IsString,
+  Max,
+  Min,
+} from 'class-validator';
+
+export class CreateCarDto {
+  @IsString()
+  @IsNotEmpty()
+  marca: string;
+
+  @IsString()
+  @IsNotEmpty()
+  modelo: string;
+
+  @IsNotEmpty()
+  @IsInt()
+  @Min(2015)
+  @Max(2023)
```

Comment:
You can store these magic numbers (and the following ones) in a `constants.ts` file

```ts
export const MIN_YEAR = 2015;
export const MAX_YEAR = 2023;
```

```suggestion
  @Min(MIN_YEAR)
  @Max(MAX_YEAR)
```

Reply:
You are right, that is more clear, also i think actual year should be dynamic
```
const MIN_YEAR=2015
const ACTUAL_YEAR= new Date().getFullYear()
```